README.md

```diff
@@ -285,6 +285,32 @@ Training [CoCa](https://arxiv.org/abs/2205.01917) models is enabled through spec
 ```
 Credit to [lucidrains](https://github.com/lucidrains) for [initial code](https://github.com/lucidrains/CoCa-pytorch), [gpucce](https://github.com/gpucce) for adapting the code to open_clip, and [iejMac](https://github.com/iejMac) for training the models.
 
+### Generating text with CoCa
+
+To generate text with coca this should work
+
+```python
+import open_clip
+from PIL import Image
+
+model, _, transform = open_clip.create_model_and_transform(
+  model_name="coca_ViT-B-32",
+  pretrained="path/to/pretrained"
```

Comment:
Point to an existing model instead (see readme, the key is specified at the end)